content/en/docs/v3.6/dev-guide/api_reference_v3.md

```diff
@@ -459,16 +508,27 @@ Empty field.
 
 | Field | Description | Type |
 | ----- | ----------- | ---- |
+| (versionpb.etcd_version_msg) |  | option |
 | header |  | ResponseHeader |
 | deleted | deleted is the number of keys deleted by the delete range request. | int64 |
 | prev_kvs | if prev_kv is set in the request, the previous key-value pairs will be returned. | (slice of) mvccpb.KeyValue |
 
 
 
+##### message `DowngradeInfo` (api/etcdserverpb/rpc.proto)
+
+| Field | Description | Type |
+| ----- | ----------- | ---- |
+| enabled | enabled indicates whether the cluster is enabled to downgrade. | bool |
+| targetVersion | targetVersion is the target downgrade version. | string |
+
+
+
```

Comment:
Could you please remove these two empty lines? This is triggering the linter violation.